### DISCUSSION TITLE
FileUpload randomly fails on Laravel Vapor with higher traffic

### DISCUSSION THREAD

**Discussion by sweebee** [filamentphp/filament#14446] [Help]
### Package

Form builder

### Package Version

v3.2.115

### How can we help you?

I'm running my application on Laravel Vapor, and when uploading a file with the FileUpload it randomly fails. It only seems to happen if there is more traffic than normally. In the morning for example everything works fine. Things i've tried but made no difference:

- use local file upload instead of S3
- switch between octane on/off

If I test it on the dev environment (also on Vapor) everything works fine, I guess because there is no traffic on that one. See the attached video for an example:

https://github.com/user-attachments/assets/b6e09f81-8479-4cdf-ac95-4d7c84dc85ac

**Comment by fagnerdireito** [comment]
i have the same.

when i try upload multiples files return error on console.

![image](https://github.com/user-attachments/assets/51fea3e6-c160-4285-9982-1654c39e3333)

my code 

![image](https://github.com/user-attachments/assets/06e92785-fc54-4174-ace9-cdcac31eb801)


When I upload only one file, it works, but when I add two or more files at once, the modal closes unexpectedly.
